api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -156,3 +156,43 @@ enum IneligibleStatus {
   // Only applicable to members.
   INELIGIBLE_STATUS_EXPIRED = 4;
 }
+
+// Review is a review of an access list.
+message Review {
+  // header is the header for the resource.
+  teleport.header.v1.ResourceHeader header = 1;
+
+  // spec is the specification for the access list review.
+  ReviewSpec spec = 2;
+}
+
+// ReviewSpec is the specification for an access list review.
+message ReviewSpec {
+  // access_list is the name of the access list that this review is for.
+  string access_list = 1;
+
+  // review is the user who performed the review.
+  string reviewer = 2;
+
+  // review_date is the date that this review was created.
+  google.protobuf.Timestamp review_date = 3;
+
+  // body is an optional plaintext attached to the review that can be used by the review for arbitrary
+  // reporting.
+  string body = 4;
+
+  // changes are the changes made as part of the review.
+  ReviewChanges changes = 5;
+}
+
+// ReviewChanges are the changes that were made as part of the review.
+message ReviewChanges {
+  // audit_frequency_changed is populated if the audit frequency was changed.
+  google.protobuf.Duration frequency_changed = 1;
+
+  // membership_requirements_changed is populated if the requirements were changed as part of this review.
+  AccessListRequires membership_requirements_changed = 2;
+
+  // removed_users contains the users that were removed as part of this review.
+  repeated string removed_users = 3;
```

Comment:
`removed_members` instead? only members can be removed during this review right?

Reply:
Thank you, that's right. Good catch.